network/definition.yaml

```diff
@@ -6,15 +6,28 @@ metadata:
     upbound.io/ui-schema: |-
       ---
       configSections:
+      - title: Network Info
+        description: Information about this network fabric
+        items:
+        - name: id
+          controlType: singleInput
+          type: string
+          path: ".spec.id"
+          title: Network ID
+          description: Network ID that other objects will use to refer (and connect) to this network fabric
+          default: network
+          validation:
+          - required: true
+            customError: Network ID is required.
       - title: Cluster Info
```

Comment:
the clusterRef field in the Network is required (right now at least), because the cluster ID gets embedded into some tags on the subnets inside the Network composition.  That is how k8s discovers what subnets it can use: https://docs.aws.amazon.com/eks/latest/userguide/network_reqs.html#vpc-subnet-tagging

I don't think we support more than 1 cluster on a single network (right now at least).  We may be able to figure that out with more research, but for this first release it's only 1 cluster per network.

note that It's a bit hacky here too because this is a bit of a circular reference, i.e. the network needs to know about the cluster and the cluster needs to know about the reference.  Perhaps this can be improved when we iterate, but this release that is **finally** working after a lot of effort will ship this way.